index.html

```diff
@@ -200,20 +238,33 @@ <h5>Languages</h5>
                 id="name"
                 minlength="3"
                 maxlength="30"
-                name="name"
+                name="fname"
                 max-length="3"
-                placeholder="Full name"
+                placeholder="First name"
               />
             </div>
+
             <div class="form-control">
+              <input
+                required
+                type="text"
+                id="name"
+                minlength="3"
+                maxlength="30"
+                name="lname"
+                max-length="3"
+                placeholder="Last name"
+              />
```

Comment:
- [ ][OPTIONAL] For the form element as mentioned din the project requirements :point_down: 
![image](https://user-images.githubusercontent.com/97521557/229078440-68a29897-feaa-4a4c-b23f-90ba38da8848.png)
There should be only one field, for full name but since you have used two fields on the mobile version as well I am adding it as Optional. but I think having a single field is better user experience